mqtt/mqtt-bridge/src/client.rs

```diff
@@ -183,29 +185,28 @@ impl BridgeIoSource {
 }
 
 /// This is a wrapper over mqtt3 client
-pub struct MqttClient<T>
+pub struct MqttClient<H>
```

Comment:
why are you changing the name for the generic type? I think T is standard value for generic types

Reply:
In C# - yes. In rust usually - the first letter of the trait

Reply:
According to Rust Programming Language book, T is the default.